app/lib/exchange-rate/exchange-rate-service.ts

```diff
@@ -66,4 +67,24 @@ class ExchangeRateService {
   }
 }
 
+export const serializeRates = (rates: Rates): SerializedRates => {
+  const { timestamp, ...rest } = rates;
+  return {
+    ...Object.fromEntries(
+      Object.entries(rest).map(([key, value]) => [key, value.toString()]),
+    ),
+    timestamp: timestamp.toString(),
```

Comment:
any thoughts on separating out timestamp from the rest of the Rates object even though we call toString on both? This seems more clear to me that we are handling two different types of data